OneCommunityGlobal/HGNRest
# Shiwani Make permission for "Total Valid Weekly Summaries" number editable

# Description
<img width="724" alt="Screenshot 2023-10-19 at 1 37 12 PM" src="https://github.com/OneCommunityGlobal/HGNRest/assets/78906820/c1ca7c75-1f6d-4704-8c69-774556cc1c29">


## Related PRS (if any):
To test this backend PR you need to checkout the #[1459](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1459) frontend PR.
…

## Main changes explained:
- Added 'totalValidWeeklySummaries' to Owner and Administrator in createInitialPermissions.js

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Follow instructions in frontend PR

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HGNRest/assets/78906820/85989b0f-8b19-409d-b747-5e700944328f



## Note:
Include the information the reviewers need to know.
